code/workspaces/web-app/src/components/stacks/StackCards.js

```diff
@@ -24,36 +24,53 @@ const styles = theme => ({
   },
 });
 
+export const sortStacksByDisplayName = (a, b) => {
```

Comment:
I'm not sure you need to write the sort function yourself given we have a dependency on lodash. I think something like this should work

```
sortedStacks = sortBy(stacks, (stack) => stack.displayName.toLowerCase();
```

Reply:
Good shout, I will make the change. Thanks!